### PR TITLE
feat: Include children `controllers` and `integration` folders into minitest paths

### PR DESCRIPTION
**Pull request by torchello**
@ptrdvrk found an app where tests are located in `test/internal/app/controllers`. We can easily include children dirs into the list of possible paths.

### CODE REVIEW

**Review by apotterri — CHANGES_REQUESTED** [review]
I don't see any tests for `service`? This might be a good time to add some....

**Comment by torchello** [comment]
@apotterri The services are mostly covered by integration tests for init/status commands. It's tricky to cover them by unit tests, but I agree we should have at least some smoke specs. Please check again.

File: spec/service/integration_test_path_finder_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'appmap/service/integration_test_path_finder'
+
+describe AppMap::Service::IntegrationTestPathFinder do
+  subject { described_class }
+
+  describe '.count' do
+    it 'counts existing paths' do
```
Comment (apotterri) [inline spec/service/integration_test_path_finder_spec.rb:10]: These tests could pass even if the code is broken, couldn't they?

Might be better to use something like https://github.com/fakefs/fakefs to verify that `service` is really working as expected.

**Review by apotterri — CHANGES_REQUESTED** [review]

File: spec/service/integration_test_path_finder_spec.rb (hunk 1 above)
Comment (torchello) [reply spec/service/integration_test_path_finder_spec.rb:10]: @apotterri I adjusted the service so it can be tested with unit tests. Please have a look.

**Review by apotterri — APPROVED** [review]
Looks good, thanks.

**Pull request merged**